src/app.js

```diff
@@ -505,6 +505,19 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
     }
   }, false)
 
+  window.addEventListener('keypress', (e) => {
```

Comment:
That code should be put in a separate handler (class), so we don't spam `app.js`.
can be in another PR though

Reply:
ok, will do